public/webrtc.js

```diff
@@ -205,6 +213,8 @@ function pageReady() {
     localVideo = document.getElementById('localVideo');
     remoteVideo = document.getElementById('remoteVideo');
 
+    window.onresize = updateCSS
```

Comment:
I kept it because I added a feature that swaps the `columnDivider` and the `rowDivider` of the grid if the height of the videos container is greater than it's width, so that the layout is nicer when using a vertical screen (like a phone). And if we want the layout to be updated with this feature when the screen is resized we need it